app/assets/javascripts/discourse/tests/acceptance/user-test.js

```diff
@@ -177,3 +179,123 @@ acceptance("User - Saving user options", function (needs) {
     );
   });
 });
+
+acceptance(
+  "User - Notification level when `can_{mute|ignore}_user` are false",
+  function (needs) {
+    needs.user({ username: "eviltrout", id: 1, ignored_ids: [] });
+
+    needs.pretender((server, helper) => {
+      server.get("/u/charlie.json", () => {
+        const cloned = cloneJSON(userFixtures["/u/charlie.json"]);
+        cloned.user.can_ignore_user = false;
+        cloned.user.can_mute_user = false;
+        return helper.response(200, cloned);
+      });
+    });
+
+    test("Notification level button is not rendered", async function (assert) {
+      await visit("/u/charlie");
+      assert.notOk(exists(".user-notifications-dropdown"));
+    });
+  }
+);
+
+acceptance(
+  "User - Muting other user with notification level dropdown",
+  function (needs) {
+    needs.user({ username: "eviltrout", id: 1, ignored_ids: [] });
+
+    needs.pretender((server, helper) => {
+      server.get("/u/charlie.json", () => {
+        const cloned = cloneJSON(userFixtures["/u/charlie.json"]);
+        cloned.user.can_ignore_user = true;
```

Comment:
Do you need the `can_ignore_user` flag here? Same for `can_mute_user` for the ignore action test case below.